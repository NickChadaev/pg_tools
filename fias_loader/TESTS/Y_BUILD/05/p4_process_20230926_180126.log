2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_05 postgres /home/n.chadaev@abrr.local/Y_BUILD/05/stage_gar_c_05.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Дагестан Респ, id_region: 6, Execute: /home/n.chadaev@abrr.local/Y_BUILD/05/stage_gar_c_05.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_REESTR_OBJECTS_20230925_4accbe21-3563-4ef3-bebd-a04d80398d78.XML
[18:14:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_ADDR_OBJ_20230925_b11b14f6-9625-4040-bb5a-b569e4a1b68c.XML
[18:15:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_HOUSES_20230925_ed137c49-0e5e-47a0-b5ec-dc2219cba612.XML
[18:31:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_STEADS_20230925_346aaff2-04c1-4a72-b0cb-011eedfbd60b.XML
[18:35:14] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_APARTMENTS_20230925_12049ec7-9da2-4719-8e7f-da62feb28d6a.XML
[18:36:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_ROOMS_20230925_d996fb9b-678f-4264-bbb9-fe30f070dc28.XML
[18:36:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_CARPLACES_20230925_f5ab054c-6945-425d-a776-099999ce2f14.XML
[18:36:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_ADM_HIERARCHY_20230925_eae37549-152f-4d1d-b051-64aaa3528fe8.XML
[18:50:03] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_MUN_HIERARCHY_20230925_378983f7-8863-4397-9b26-aa5acaeeaeea.XML
[19:05:29] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_CHANGE_HISTORY_20230925_6599c0be-c3c6-4736-b933-dec8c5c17780.XML
[19:47:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_ADDR_OBJ_DIVISION_20230925_ee8348e5-d748-4e98-9a0a-9592abdede7c.XML
[19:47:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_ADDR_OBJ_PARAMS_20230925_b80a43d4-c486-4bd4-aeb2-def702e5d01f.XML
[19:52:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_STEADS_PARAMS_20230925_6253f8dc-4ae6-4eab-a44b-2316e6d1b94a.XML
[20:29:43] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_HOUSES_PARAMS_20230925_59ee3358-30cf-4755-b502-9f94c3a7e49d.XML
[22:08:32] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_APARTMENTS_PARAMS_20230925_103b77e5-f899-4984-9f82-6f4ff4edb724.XML
[22:10:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_ROOMS_PARAMS_20230925_f165ef0a-17c6-4adb-9676-c7cf1dd17d0f.XML
[22:10:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_CARPLACES_PARAMS_20230925_df1b9d22-d4e7-49ff-aefc-6199872a4210.XML
[22:10:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/05/AS_NORMATIVE_DOCS_20230925_f0d50599-9013-4df1-8e26-4ac184359814.XML
[22:12:51] ...  -- Data finish
[22:12:51] ...  -- Set LOGGED
[22:13:36] ... Stop process
--------------------------------------------------------
