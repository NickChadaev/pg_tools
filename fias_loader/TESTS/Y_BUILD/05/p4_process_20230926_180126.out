 save_gar_version 
------------------
               71
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                71
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 1071376
gar_fias.as_addr_obj: 35992
gar_fias.as_houses: 1217133
gar_fias.as_steads: 306605
gar_fias.as_apartments: 65273
gar_fias.as_rooms: 33
gar_fias.as_carplaces: 0
gar_fias.as_adm_hierarchy: 1082147
gar_fias.as_mun_hierarchy: 1212604
gar_fias.as_change_history: 3605974
gar_fias.as_addr_obj_division: 76
gar_fias.as_addr_obj_params: 354104
gar_fias.as_steads_params: 3036715
gar_fias.as_houses_params: 7955649
gar_fias.as_apartments_params: 172332
gar_fias.as_rooms_params: 31
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 171283
